velox/exec/tests/AggregationFuzzer.cpp

```diff
@@ -659,21 +697,31 @@ void AggregationFuzzer::go() {
 
 velox::test::ResultOrError AggregationFuzzer::execute(
     const core::PlanNodePtr& plan,
-    bool injectSpill) {
+    bool injectSpill,
+    bool abandonPartial) {
   LOG(INFO) << "Executing query plan: " << std::endl
             << plan->toString(true, true);
 
   velox::test::ResultOrError resultOrError;
   try {
     std::shared_ptr<TempDirectoryPath> spillDirectory;
     AssertQueryBuilder builder(plan);
+
     if (injectSpill) {
       spillDirectory = exec::test::TempDirectoryPath::create();
       builder.spillDirectory(spillDirectory->path)
           .config(core::QueryConfig::kSpillEnabled, "true")
           .config(core::QueryConfig::kAggregationSpillEnabled, "true")
           .config(core::QueryConfig::kTestingSpillPct, "100");
     }
+
+    if (abandonPartial) {
```

Comment:
I saw that testAggregations() only tests abandon-partial when grouping key is not empty, but we don't check it here. Does it matter?

Reply:
@kagamiori Good catch, Wei. I'll make a change to not test this path for global aggregations. The code path we are targeting doesn't kick in for global aggregations, hence, testing is just wasted valuable time and compute. Thanks.